### ISSUE TITLE
Changing Name of Existing Campaign Resulted in Redirect Loop Error

### ISSUE THREAD

**Issue by namimody** [DoSomethingArchive/legacy-website#5034]
Shower Songs (node 3059) was changed to Shower with Beyonce for a stunt on 9/4. The campaign name, CTA, photo, and some content was changed, and the campaign was highlighted on the homepage, but clicking on it intermittently caused: 

![image](https://cloud.githubusercontent.com/assets/12396076/9690814/a72baf96-530c-11e5-9f6b-c312ea71e353.png)

OR (see second banner notification)

![image](https://cloud.githubusercontent.com/assets/12396076/9690844/d2342b50-530c-11e5-9abe-cc5ec446b480.png)

The redirects were set up like this:

![image](https://cloud.githubusercontent.com/assets/12396076/9690848/de08e998-530c-11e5-9bfc-3b4b9854c706.png)

And deleting campaigns/shower-songs did not resolve the issue, and it did not seem as thought any other redirects were set up for this node.

For this example, we just changed the name back to Shower Songs, and the error disappeared, but we need to figure out if this happens when changing the name of any campaign.

**Comment by mshmsh5000** [comment]
Drupal is sometimes overly clever when you update a content title -- it updates the URL based on the title, and then creates a redirect from the old URL to the new URL.

The problem we had in the past was update a campaign title would (a) update the campaign URL, and (b) create a redirect from the old URL to the new one. Manually created redirects would sometimes cause a redirect loop by redirecting to the old URL.

I'm not going to try to recreate the issue on prod, but the best place to look is the "URL redirects" tab on the node edit form itself:

![node-redirects](https://cloud.githubusercontent.com/assets/149811/9691156/c24d09c6-530e-11e5-810a-8880e3cb3ed8.png)

**Comment by namimody** [comment]
Thanks @mshmsh5000!! I chatted about this a bit with @mikefantini as well, and it seems this issue is pretty specific to this campaign and instance. It seems that the campaign name was changed twice -- once to say "Beyonce" and once to say "Beyoncé" and since the site can't distinguish between the URLs having an e with an accent vs. just an e, the change created a redirect loop. We determined that to prevent this from happening again, leads who change the name of a campaign should check there is no redundancy between the URL path settings (the alias) and the URL redirects. Talking about this at the next campaigns/tech sync -- closing issue.
